cmd/juju/waitfor/waitfor.go

```diff
@@ -19,6 +22,7 @@ defined by the supplied query.
 Examples:
 	juju wait-for unit mysql/0
 	juju wait-for application mysql --query='name=="mysql" && (status=="active" || status=="idle")'
+	juju wait-for model default --query='forEach(units, unit => startsWith(unit.name, "ubuntu"))'
```

Comment:
It'd be handy to have a description for the example of what it does

Reply:
Yeah, I've got a whole ticket to now go and fill in the documentation.